overlord/state/taskrunner_test.go

```diff
@@ -93,24 +93,24 @@ func ensureChange(c *C, r *state.TaskRunner, sb *stateBackend, chg *state.Change
 //     <task>:...:1,2         - one of the above, and add task to lanes 1 and 2
 //     chg:abort              - call abort on the change
 //
-// Task wait order: ( t11 | t12 ) => ( t21 ) => ( t31 | t32 )
+// Task wait order: ( t11 | t12 ) => ( t21 ) => ( t22 ) => ( t31 | t32 ) => ( t23 )
```

Comment:
I think the original intention was for each => level to have the same prefix number:

( t11 | t12 ) => ( t21 ) => ( t31 ) => ( t41 | t42 ) => ( t51 )

this need more renumbering though :(